content/tutorials/editors.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Recommended Editors
+---
+
+To edit your code, you'll need an editor.
+
+{{% notice info %}}
+To write code for the kit, you don't need anything specific in your editor, you just need to be able to edit Python (`.py`) files, and get them onto a USB drive.
+{{% /notice %}}
+
+Any editor will do, so it's recommended to use one you're already familiar with. If you don't already have a preference, here's some great ones, available on Windows, macOS, and Linux!
```

Comment:
Is that the right capitalisation of macOS (I was expecting MacOS, but that's based on English grammar rules rather than Apple)

Reply:
i'm afraid it is. https://www.apple.com/macos/